radicle-tracker/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+use std::hash;
+
+pub mod thread;
+use thread::*;
+
+pub struct Issue<Author> {
+    pub author: Author,
+    pub title: String,
+    pub thread: Thread<Comment<Author>>,
+    pub meta: (), // TODO(fintan): fill in meta data
```

Comment:
Do you need input on what could go in here?

Reply:
I think I have them documented in https://radicle.community/t/cocoda-issues-codecollab/114 so if there's anything missing there lemme know :ok_hand: